src/miniwiki.rs

```diff
@@ -284,4 +291,14 @@ fn render_arguments(parameters: &Vec<DumpFunctionParameter>) -> Fragment {
                 }
             </div>
         )))
+}
+
+fn member_element_class<'a>(tags: &'a BTreeSet<String>, main_class: &str) -> String {

```

Comment:
```suggestion
fn member_element_class(tags: &BTreeSet<String>, main_class: &str) -> String {
```

Shouldn't need lifetime annotations!